content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -0,0 +1,119 @@
+ 
+ 
+[1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
+[2]: https://app.datadoghq.com/account/settings#api
+[3]: /getting_started/site/
+[4]: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html
+{{% /tab %}}
+{{% tab "AWS Console" %}}
+ 
+If you want to set up metric streams using the AWS Console, follow these steps for each AWS region.
+ 
+1. Create a new Kinesis Data Firehose delivery stream with the following specifications:
+ - For source, select “Direct PUT or other sources”
+- For destination:
+  - Select Third Party Destination: `Datadog`.
+  - Select your [Datadog site][4] under metrics: `Datadog US` or `Datadog EU`.
+  - For access key, enter your [Datadog API key][1].
+   - For retry duration, enter `60 seconds`.
+   - For S3 backup, select `Failed data only` and choose the desired S3 bucket for backup.
+ - For HTTP endpoint buffer conditions:
+   - Enter `4MB` for buffer size, and `60 seconds` for buffer interval.
+ - For S3 buffer conditions:
+   - Enter `4MB` for buffer size, and `60 seconds` for buffer interval.
+ - For S3 compression, choose `GZIP`.
+ - Enable error logging.
+2. Create your [CloudWatch Metric Stream][2] with the following steps:
+ 1. Choose whether you want to stream all CloudWatch metrics, or choose specific namespaces with “Include” or “Exclude” lists.
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/cloudwatch-metric-stream.png" alt="Metric streams selection tab" responsive="true" style="width:60%;">}}
+ 2. Select the Firehose you created in Step 1 to use for sending the metrics to Datadog.
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/firehose.png" alt="Metric streams selection tab" responsive="true" style="width:60%;">}}
+ 3. Create a new service role to put records in Kinesis Data Firehose.
+ 4. Choose OpenTelemetry 0.7 as the output format.
+ 5. Name your metric stream.
+ 6. Click **Create metric stream**.
+ 
+### Results
+ 
+Once you see the Metric Stream resource has been successfully created, wait five minutes for Datadog to recognize this. Then go to the [Datadog AWS Integration tile][3] to see this is working by viewing the "CloudWatch Metric Streams" tab for the specified AWS account.
+ 
+{{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/active-regions.png" alt="Metric streams selection tab" responsive="true" style="width:60%;">}}
+**Important Note**: If you've already enabled polling CloudWatch APIs, the transition to streaming could cause a brief (up to five minutes) period where the specific metrics you are streaming are double-counted in Datadog. This is because of the difference in timing between when Datadog’s crawlers are running and submitting your CloudWatch metrics, and when Datadog recognizes that you have started streaming those metrics and turn off the crawlers.
```

Comment:
```suggestion
**Note**: If you've already enabled polling CloudWatch APIs, the transition to streaming could cause a brief (up to five minutes) period where the specific metrics you are streaming are double-counted in Datadog. This is because of the difference in timing between when Datadog’s crawlers are running and submitting your CloudWatch metrics, and when Datadog recognizes that you have started streaming those metrics and turn off the crawlers.
```